challenges/repeatedWord/repeated-word.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-useless-escape */
+function firstRepeatedWord(str) {
+  const wordArray = str.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g, '').split(' ');
+  const tempArray = [];
+  let word = '';
+  let i = 0;
+  while(word === '') {
+    if(tempArray.includes(wordArray[i].toLowerCase())) { word = wordArray[i]; }
+    tempArray.push(wordArray[i].toLowerCase());
+    i++;
+    if(i >= wordArray.length) word = 'no duplicates found';
+  }
+  return word;
+}
```

Comment:
you could do this in a somewhat more concise way with this (mostly the same as what you have but with a for loop at the end.:
function repeatedWord(str) {
  const arr = str.toLowerCase().replace(/[.,#!$%^&*;:{}=\-_`~()]/g, '').split(' ');
  const obj = {};
  for(let i = 0; i < arr.length; i++) {
    if(obj[arr[i]]) return arr[i];
    else obj[arr[i]] = 1;
  }
  return null;
}